common/org.eclipse.tracecompass.incubator.releng-site/category.xml

```diff
@@ -84,6 +84,8 @@
    <bundle id="org.eclipse.tracecompass.incubator.tmf.ui.multiview.ui" version="0.0.0">
       <category name="analyses"/>
    </bundle>
+   <bundle id="org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core"/>
```

Comment:
Should it have a category?

Reply:
Yes, I could add a category. Not sure if it could be used when installed in an IDE or as Add-on from the Trace Compass update site